### PR TITLE
Ernestomr87 On-boarding

### PR DESCRIPTION
**Pull request by stdevGeekmidget**
[mobilewallet] refs#29 On-boarding " Set/Confirm Pin"

### CODE REVIEW

**Comment by stdevGeekmidget** [comment]
refs#30 skycoin/skycoin-mobilewallet On-boarding "Create/Load Wallet" screen
refs#29 skycoin/skycoin-mobilewallet On-boarding " Set/Confirm Pin"

File: src/pages/pincode/pincode.html [hunk 1]
```diff
@@ -1,58 +1,139 @@
...
+            <div class="pin-instructions" *ngIf="status === 3">
+              <h2>Confirm PIN #</h2>
+              <p>Enter your pin once more to continue.</p>
+            </div>
+            <div id="logo" *ngIf="status < 2">
+              <img src="assets/logo/logo-white.png">
+            </div>
+          </div>
 
-      <div id="fields" [ngClass]="{'miss': showError}">
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.html:26]: Add spaces inside objects { 'miss': showError }

File: src/pages/pincode/pincode.html [hunk 2]
```diff
@@ -1,58 +1,139 @@
...
+        <div class="cell-header">
+          <div class="pin-instructions" style="margin-top: 20px;">
+            <h2>Create a wallet</h2>
+            <p>If you don’t yet have a wallet, generate a seed and we’ll create one for you. If you already have a
+              wallet, enter
+              the seed below and load it.</p>
+          </div>
+          <div class="cell-body">
+            <div class="-toggle">
+              <button class="new" ion-button round >New</button>
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.html:113]: Don't leave empty spaces at end of html tags

File: src/pages/pincode/pincode.ts [hunk 3]
```diff
@@ -1,29 +1,61 @@
...
+  seed: string;
+  confirmSeed: string;
   correct: string;
-  pin = '';
-  showError = false;
-  storageAvailable = true;
+  pin: string;
+  showError: boolean;
+  display: boolean;
+  storageAvailable: boolean = true;
```
Comment (samuelvisscher) [inline src/pages/pincode/pincode.ts:24]: No need to specify boolean, will default

File: src/match.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import {AbstractControl} from '@angular/forms';
```
Comment (samuelvisscher) [inline src/match.ts:1]: Add spaces inside ES6 imports

**Review by samuelvisscher — APPROVED** [review]
Looks good, haven't been able to test it, but let's merge to dev and create issues if anything comes up. Please take a look at the small feedback I have provided, for next PR's

**Pull request merged**